pkg/daemon/ceph/agent/agent.go

```diff
@@ -119,14 +119,13 @@ func (a *Agent) Run() error {
 	sigc := make(chan os.Signal, 1)
 	signal.Notify(sigc, syscall.SIGTERM)
 	for {
```

Comment:
We don't need the `for` loop anymore, right?

Reply:
yes, Done.